### ISSUE TITLE
Where is the option to having fairy rewards only shuffled between the fountains and not globally (in the dev build)?

### ISSUE THREAD

**Issue by Kongolan** [AmazingAmpharos/OoT-Randomizer#156]
Why can't i set that i need to visit the fairy fountains, to obtain these strong rewards (but on random, where which reward is). I don't want to find magic in a random chest somewhere. Older versions of the randomizer had this, so why not flag for it??

**Comment by phixtyseven** [comment]
I would agree that it was kind of neat knowing all fountains contain magic rewards, but it seems like a niche preference. I'd like to try to add support for specialized configurations in the future but I don't think we're quite there just yet.

**Comment by Kongolan** [comment]
If you can clarify for me, what exactly was the change, which added fountains the general item pool, then i might can look for it myself and create a pull request for it. I didn't look much into the code, but when the fountain thing got changed and maybe it's possible to split the arrays.

**Comment by phixtyseven** [comment]
Well, after a hasty peek through the commits...
* 782134e seems to introduce great fairy shuffling
* aba9184 seems to move them to the general item pool
There may be more to it, but that should be a start
